rfc/src/rfcs/0012-list.md

```diff
@@ -132,6 +152,30 @@ For example:
 }
 ```
 
+#### Output Schema
```

Comment:
I also would prefer adding this in the documentation rather than the RFC since it will evolve. Otherwise, just mention that this will be the initial version.

Reply:
I agree documentation is better--going to delete both output schemas.